tests/autobahn/Dockerfile.aiohttp

```diff
@@ -0,0 +1,18 @@
+FROM python:3.9.1
```

Comment:
We could probably use something newer, right?

Reply:
I changed it to python:3.9.5